hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2819,6 +2819,15 @@
     </description>
   </property>
 
+  <property>
+    <name>ozone.om.enable.key.path.lock</name>
```

Comment:
Please rename this property `ozone.om.enable.key.path.lock` to` ozone.om.key.path.lock.enabled`.

Also, please rename the corresponding items
-> `OZONE_OM_ENABLE_KEY_PATH_LOCK` to `OZONE_OM_KEY_PATH_LOCK_ENABLED`.

-> `getKeyPathLock() `to `getKeyPathLockEnabled()`

Reply:
I have made the changes, thank you for the comment!